README.md

```diff
@@ -58,6 +58,40 @@ long_line_under_100_chars
 # Extra Specs Coding Style
 
 - Add a blank line between `let` and `before`.
+- Group specs around the described object's public methods. Use `describe "#method_name` for instance methods and `describe ".method_name"` for class methods.
+- Context setup does not belong to the `it` block, that's the place for the assert.
+- Context description should always start like `context "when ..."` or `context "and ..."`
```

Comment:
Why not `context "with"`? 😄 
I think we cant use a less-restrictive style by allowing `grammatical relative clauses introduced by a pro-form` as a context. 
Or we keep this codding-style => http://www.betterspecs.org/#contexts

Reply:
Right, `with` is okay too. I don't understand the second sentence.

Reply:
I was trying to say that the rule is too restrictive when taken literally. :)
I think it's also redundant with this: http://www.betterspecs.org/#contexts